fala/apps/adviser/views.py

```diff
@@ -22,6 +26,8 @@ def get(self, request, *args, **kwargs):
                 "current_url": current_url,
                 "GOOGLE_MAPS_API_KEY": settings.GOOGLE_MAPS_API_KEY,
                 "LAALAA_API_HOST": settings.LAALAA_API_HOST,
+                "FEATURE_FLAG_NO_MAP": settings.FEATURE_FLAG_NO_MAP,
```

Comment:
As previously noted, we probably shouldn't expose this to the template (and I think its unused)

Reply:
have now removed.